pillars/3_HST/ccengine.sls

```diff
@@ -0,0 +1,14 @@
+ccengine:
+  # set the default here (may be overridden by subsequent sls)
+  branch: master
+mounts:
+  - spec: /dev/nvme1n1
```

Comment:
Does this device ID need to be consistent over reboots?  The reason I ask is that the device ID style used here may change upon reboot if, e.g., a new device is added, or the Linux kernel version changes.  I've found it's more consistent and reliable to refer to devices by their UUID (perhaps something like `/dev/disk/by-uuid/<UUID>` in this context) whenever you need to have consistent remounting after reboots.

Reply:
Fair enough!